x/account/keeper/coordinator.go

```diff
@@ -0,0 +1,114 @@
+package keeper
+
+import (
+	"encoding/binary"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/spn/x/account/types"
+	"strconv"
+)
+
+// GetCoordinatorCount get the total number of TypeName.LowerCamel
```

Comment:
`TypeName.LowerCamel` seems like we have a templating issue here 😄 We must create an issue for it

Reply:
fixed, and issues created
https://github.com/tendermint/starport/issues/1390